src/adaptors/null-articles.ts

```diff
@@ -0,0 +1,35 @@
+import { JsonLdObj, Iri } from 'jsonld/jsonld-spec';
+import Articles from './articles';
+
+export default class NullArticles implements Articles {
+  [Symbol.iterator](): Iterator<JsonLdObj> {
+    return {
+      next(): IteratorResult<JsonLdObj> {
+        return {
+          done: true,
+          value: undefined,
+        };
+      },
+    };
+  }
+
+  async count(): Promise<number> {
+    return 0;
+  }
+
+  async get(id: Iri): Promise<JsonLdObj> {
+    throw new RangeError(`An article with the ID ${id} could not be found`);
+  }
+
+  async set(node: JsonLdObj): Promise<void> {
+    throw new Error('Unable to add an article');
+  }
+
+  async delete(id: Iri): Promise<void> {
+
```

Comment:
In this situation, I would have a `Do nothing` comment to make it explicit.

Reply:
Resolving as we have removed the NullArticles class for now.